frontend/src/components/ContentPanel/ContentPanelPrompts/index.tsx

```diff
@@ -32,6 +34,16 @@ export const ContentPanelPrompts: React.FC<ContentPanelPromptsProps> = ({
         );
     };
 
+    // Updates the input when user stops writing
```

Comment:
This would probably be useful (and maybe a bit clearer) as a separate useDebounce hook.